kitsune/questions/views.py

```diff
@@ -580,6 +574,9 @@ def aaq(request, product_key=None, category_key=None, step=1):
                 product_config=product_config,
             )
 
+            if form.cleaned_data.get('is_spam'):
+                _save_and_mark_as_spam(request, question)
```

Comment:
I would remove the `save` part from here and keep only the spam handling. Moreover that should be renamed to something else as it essentially moves the reply to the moderation queue. It doesn't mark the reply as spam.